src/Microsoft.Azure.WebJobs.Host/Executors/TriggeredFunctionExecutor.cs

```diff
@@ -14,12 +14,14 @@ internal class TriggeredFunctionExecutor<TTriggerValue> : ITriggeredFunctionExec
         private FunctionDescriptor _descriptor;
         private ITriggeredFunctionInstanceFactory<TTriggerValue> _instanceFactory;
         private IFunctionExecutor _executor;
+        private readonly IRetryManagerProvider _retryManagerProvider;
```

Comment:
This will only provide retries for triggered functions. I think this means that you won't currently get retries for JobHost.CallAsync direct invocations, which happen for manually triggered functions as well as for http functions in AzureFunctions. I think you need to hoist the retry stuff up higher, into FunctionExecutor itself. Also add tests verifying this.

Reply:
Moved to FucntionExecutor. Please take a look.